Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.yml

```
@@ -4,18 +4,18 @@ commonfields:
 name: Recorded Future Identity
 display: Recorded Future Identity
 category: Data Enrichment & Threat Intelligence
-image:
 description: Recorded Future Identity Integration that provides access to Recorded Future Identity module data.
 configuration:
 - display: Server URL (e.g., https://api.recordedfuture.com/gw/xsoar-identity)
   name: server_url
   defaultvalue: https://api.recordedfuture.com/gw/xsoar-identity
   type: 0
   required: true
-- display: API Token
-  name: token
-  type: 4
+- name: token
+  type: 9
   required: true
+  displaypassword: API Token
+  hiddenusername: true
```

```suggestion
- display: API Token
  name: token
  type: 4
  required: false
  hidden: true
- displaypassword: API Token
  name:  token
  type: 9
  required: false
  hiddenusername: true
```
Please keep the type 4 for backward compatibility.